pkg/kube/job.go

```diff
@@ -73,7 +73,8 @@ func NewJob(clientset kubernetes.Interface, jobName string, namespace string, se
 // Create creates the Job in Kubernetes.
 func (job *Job) Create() error {
```

Comment:
Should the caller pass in the parent context through the `Create` call?

Reply:
I tried not to change public interface to decrease painful from my change to other developers. But if you recommend to pass context via `Create` method, I'll could implement this.

Reply:
I saw the discussion with Vivek. He's right about the Jobs no longer being used anywhere. This is okay for now but separately we can work on removing unused code.